src/lib.rs

```diff
@@ -343,6 +343,13 @@ pub struct TrashItemMetadata {
     pub size: TrashItemSize,
 }
 
+#[derive(Debug, Clone, Copy, Default)]
+pub enum RestoreMode {
```

Comment:
This isn't documented at all. Could you add `#![deny(missing_docs)]` to `lib.rs`? That way this would light up in CI.